### ISSUE TITLE
Add option to use black instead of white for extra vertical space

### ISSUE THREAD

**Issue by Gitoffthelawn** [diox/firefox-minimap-scroller#21]
Thank you for the excellent extension!

Could you add an option to show the extra vertical space (for short pages) as black instead of white?  This would make it clearer where the page ends, as well as avoid a large wall of glaring white area.

Thanks!

**Comment by diox** [comment]
Yep, that's a good idea, easy to implement.

**Comment by Gitoffthelawn** [comment]
Thanks!

**Comment by Gitoffthelawn** [comment]
Perhaps allow the user to specify the color via a hex string.

**Comment by diox** [comment]
Yeah, I'll implement a simple setting allowing you to specify a css color string.

**Comment by diox** [comment]
#22 implements this. Can you by any chance test it on your local machine and tell me if it works as expected for you? It it does I'll look at doing a release with it early next week.
